app/templates/profile.html

```diff
@@ -144,6 +149,29 @@ <h5 class="modal-title">My Telegram Id</h5>
                             </div>
                         </div>
                     </div>
+
+                    <!-- Calendar Privacy -->
+                    <div class="modal modal-sm" id="calendarPrivacyModal" tabindex="-1" data-bs-backdrop="static" aria-labelledby="updateCalendarPrivacyModalLabel" aria-hidden="true">
+                        <div class="modal-dialog profile-modal-dialog profile-modal-fadeIn">
+                            <div class="modal-content">
+                                <div class="modal-header profile-modal-header">
+                                    <h5 class="modal-title">Set your calendar privacy</h5>
```

Comment:
Is this a 5th level element?

Reply:
I just wrote it according to the other existing options in the profile update menu and thisis how the other ones are built. I didn't want to affect the CSS so I didn't change anything

Reply:
Yeah I see there is some HTML in our project using `h5` because of the font size :( That's not legit, probably I've missed it in previous PRs.

Can you please change fix it just here?

Reply:
ok should I change it to h1? h2? and set the CSS accordingly?

Reply:
If it's a title, you can change it to the highest `h` that doesn't exist in the page.
If not, you can change it to label/summary/another proper element, and to set the CSS accordingly :)

Reply:
Done !
Changes it to h1 (as it is a sort of title for the ard) and updated the css accordingly :)